stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1771,6 +1892,20 @@ func _TFCOpSetAttrString(_ op: CTFEOp,
   }
 }
 
+/// Wrapper around TFE_OpSetAttrString that handles converting the Swift Stdlib
+/// String into a buffer that TFE_OpSetAttrString can read.
+@usableFromInline
+@_silgen_name("_swift_tfc_OpSetAttrFunctionName")
+func _TFCOpSetAttrFunctionName(_ op: CTFEOp,
+                               _ attrName: UnsafePointer<Int8>,
+                               _ value: String) {
+  value.utf8CString.withUnsafeBufferPointer { buffer in
+    // utf8CString is null-terminated; TFE_OpSetAttrString wants
```

Comment:
s/TFE_OpSetAttrString/TFE_OpSetAttrFunctionName/

(also in the doc comment above this function)

Reply:
Thanks for catching this, done.